src/server/bg_services/namespace_monitor.js

```diff
@@ -111,6 +111,31 @@ class NamespaceMonitor {
         });
     }
 
+    /**
+     * When using aws sdk v3 we get the error object with values that start with an uppercase instead of lowercase
+     * we want to fix this value to be start with a lowercase.
+     * 
+     * @param {object} err
+     */
+    _fix_error_object(err) {
+        for (const key in err) {
+            if (Object.hasOwn(err, key)) {
+                // Currently we want to fix only the value "Code",
+                // if we will want to fix all the values we can remove this "if" statement
+                if (key === "Code") {
+                    const lowercaseKey = key.toLowerCase();
+                    // If we remove the "if" statement above (key === "Code") then we need to check also key !== lowercaseKey
+                    // to avoid running over a valid value.
+                    // remove the next comment and delete the if 2 lines below. 
+                    // if (key !== lowercaseKey && !Object.hasOwn(err, lowercaseKey)) {
+                    if (!Object.hasOwn(err, lowercaseKey)) {
+                        err[lowercaseKey] = err[key];
+                    }
+                }
+            }
+        }
+    }
+
```

Comment:
This is a general function, so it should be moved to the file:
src/sdk/noobaa_s3_client/noobaa_s3_client.js

Reply:
Sure